examples/configuring/README.md

```diff
@@ -0,0 +1,109 @@
+Configuring GraphDB
+===
+
+This document provides detailed instructions on how to configure GraphDB, 
+including setting properties, secret properties, and additional configurations. 
+
+It covers various methods to manage configuration options, such as using 
+ConfigMaps and Secrets, and highlights some good practices to ensure secure 
+and efficient setup. Additionally, it explains how to set Java arguments 
+and environment variables for GraphDB.
+
+All GraphDB configuration options can be found 
+[here](https://graphdb.ontotext.com/documentation/10.0/configuring-graphdb.html#configuration-properties).
+
+## Properties
+
+This section is used to set environment variables directly from the `values.yaml` file. 
+The configurations will be stored in a ConfigMap. These properties typically include 
+non-sensitive informatin, such as product settings.
+
+```yaml
+configuration:
+  properties:
+    graphdb.workbench.importDirectory: "/opt/graphdb/home/graphdb-import"
+    graphdb.cluster.sync.timeoutS: 600
+    graphdb.workbench.maxConnections: 10
+```
+
+## Secret Properties
+
+This section is used to set secret properties directly from the `values.yaml` file.
+The configuration will be stored in a Kubernetes Secret.
+These properties typically include sensitive information, such as passwords or API keys.
+
+```yaml
+configuration:
+  secretProperties:
+    graphdb.connector.keystorePass: "xxxx"
+```
+
+**Warning: This method of configuring GraphDB is strongly discouraged, as it may 
```

Comment:
Mention which is the proper approach (`configuration.extraProperties.existingSecret` & `extraEnvFrom`/`extraEnv`)